src/components/forms/SnapshotForm.tsx

```diff
@@ -1,59 +1,34 @@
-import React, { FC, KeyboardEvent } from "react";
+import React, { KeyboardEvent } from "react";
 import {
   Button,
   Col,
   Form,
-  Icon,
   Input,
-  List,
   Modal,
   Row,
-  Tooltip,
 } from "@canonical/react-components";
 import { getTomorrow } from "util/helpers";
 import SubmitButton from "components/SubmitButton";
-import { TOOLTIP_OVER_MODAL_ZINDEX } from "util/zIndex";
 import { SnapshotFormValues } from "util/snapshots";
 import { FormikProps } from "formik/dist/types";
-import NotificationRow from "components/NotificationRow";
 
 interface Props {
   isEdit: boolean;
-  formik: FormikProps<SnapshotFormValues>;
+  formik: FormikProps<
+    SnapshotFormValues<{ stateful?: boolean; description?: string }>
```

Comment:
I think we don't need the description here. Probably better to remove it.

Reply:
description is available for custom storage volume snapshots but not instance snapshot. Since this component is used for both creating instance and custom volume snapshots, the form may contain description field

Reply:
I am not sure the description on the snapshot is useful. Maybe we should streamline and leave it out. Wdyt?

Reply:
Yes I agree, description is even less meaningful for scheduled snapshots. Maybe we can leave it out for now and add it if it turns out to be a super popular thing